docs/hive-integration/kube-api-getting-started.md

```diff
@@ -0,0 +1,551 @@
+    "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+    "event_time": "2021-06-28T21:42:20.768Z",
+    "message": "Generated image (Image type is \"minimal-iso\", SSH public key is not set)",
+    "severity": "info"
+  },
+```
+
+### Boot The Host From The Discovery Image
+* The procedure described below is called: **Boot It Yourself**.
+* There is an alternative automated procedure using `BMH`. For details, check the [Zero Touch Provisioning (ZTP) advanced section](#advanced-zero-touch-provisioning).
+
+#### 1. Download the image
+* Minimal-ISO is the default image type here; therefore, expect an image at the size of ~100MB.
+
+```bash
+curl --insecure -o image.iso $(kubectl -n demo-worker4 get infraenvs.agent-install.openshift.io myinfraenv -o=jsonpath="{.status.isoDownloadURL}")
+```
+
+#### 2. Boot the host from image
+* Use virtual media / USB drive / other methods to boot your host using the discovery image.
+
+### 3. Host discovery and approval
+* The Agent CRD represents a Host that boot from a discovery image and registered to a cluster, and created automatically by assisted-service. [Find more details about the agent CRD here](README.md#agent).
+* At first, it will look as follows:
+```bash
+kubectl -n demo-worker4 get agents
+```
+```bash
+NAME                                   CLUSTER        APPROVED
+07e80ea9-200c-4f82-aff4-4932acb773d4   test-cluster   false
+```
+* To trigger the installation, you will need to approve the agent
+
+```bash
+kubectl -n demo-worker4 patch agents.agent-install.openshift.io 07e80ea9-200c-4f82-aff4-4932acb773d4 -p '{"spec":{"approved":true}}' --type merge
+```
+* You should now see it got approved
+```bash
+NAME                                   CLUSTER        APPROVED
+07e80ea9-200c-4f82-aff4-4932acb773d4   test-cluster   true
+```
+
+### 4. Cluster Installation
+* Installation has started, to monitor the progress you may inspect cluster events:
+```bash
+curl -s -k $(kubectl -n demo-worker4 get agentclusterinstalls.extensions.hive.openshift.io test-agent-cluster-install -o=jsonpath="{.status.debugInfo.eventsURL}")  | jq "."
+```
+* Another option is to inspect `AgentClusterInstall` conditions, [as mentioned above](#AgentClusterInstall-resource-conditions).
+
+### 5. End Result
+* Lastly, when installed successfully:
+```json
+{
+  "cluster_id": "02a89bb9-6141-4d14-a82e-42f254217502",
+  "event_time": "2021-06-29T16:45:21.643Z",
+  "message": "Successfully finished installing cluster test-cluster",
+  "severity": "info"
+}
+```
+
+## Advanced: Zero Touch Provisioning
+* The procedure described below is called: **Zero Touch Provisioning (ZTP)**.
+* There is an alternative manual procedure, not involving `BMH`. For details, check the [Boot The Host From The Discovery Image](#boot-the-host-from-the-discovery-image).
+
+### Utilize BareMetalHost
+
+* Examples below generated by [dev-scripts](https://github.com/openshift-metal3/dev-scripts/) and were modified manually.
+
+#### 1. Create BMC Secret
```

Comment:
It would be good to expand the BMC acronym for anyone not familiar.

Reply:
Done